collectd_mesos_plugin.py

```diff
@@ -215,8 +215,10 @@ def run(self):
                 task = value.replace(".", "_")
             if name == 'SHUB_JOBKEY':
                 kumo_job = value
+            if name == 'MARATHON_APP_DOCKER_IMAGE':
+                docker_image = value
 
-        if kumo_job:
+        if kumo_job and "kumo" in docker_image:
```

Comment:
This naive check was detecting a DS job as Kumo and was failing miserably
trying to get kumo details later. This prevented the collection of data on
the node where that DS job was running.

We can change the check to just `if kumo_job and not docker_image`, what do
you think?

On Mon, Sep 5, 2016 at 08:36 Daniel Graña <notifications@github.com> wrote:

> In collectd_mesos_plugin.py
> <https://github.com/scrapinghub/marathon-apps-collectd-plugin/pull/6#discussion_r77537612>
> :
>
> >
> > -        if kumo_job:
> > +        if kumo_job and "kumo" in docker_image:
>
> @cyberplant <https://github.com/cyberplant>: what motivated this change?
>
> Kumo jobs doesn't have MARATHON_APP_DOCKER_IMAGE set, they are not managed
> by Marathon so docker_image will be always None and raises an error in
> this line.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scrapinghub/marathon-apps-collectd-plugin/pull/6/files/dd01052d1d53cd296eb6e713419e16250d4e13b2#r77537612>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABLdbMMbFeRUAqsrVgH9vyt9SNf58AWxks5qnDcQgaJpZM4JtlSF>
> .
>